db/mongodb.go

```diff
@@ -235,9 +235,11 @@ func (m *MongoDB) ExtraIndexes(idxs []string) error {
 	c := m.db.Collection(companyTableName)
 	var i []mongo.IndexModel
 	for _, v := range idxs {
-		v = fmt.Sprintf("json.%s", v)
+		fullKey := fmt.Sprintf("json.%s", v)
+		indexName := fmt.Sprintf("idx_json.%s", v)
```

Comment:
Lembre-se do estilo Go de nomear as variáveis de escopo local

```suggestion
		k := fmt.Sprintf("json.%s", v)
		n := fmt.Sprintf("idx_json.%s", v)
```